pom.xml

```diff
@@ -45,15 +45,12 @@
         <!-- JBoss dependency versions -->
         <version.jboss.maven.plugin>7.4.Final</version.jboss.maven.plugin>
 
-        <!-- Define the version of the JBoss BOMs we want to import to specify tested stacks. -->
+        <!-- Define the version of the JBoss EAP BOM we want to import to specify tested stacks. -->
         <version.org.jboss.bom.eap>6.4.2.GA</version.org.jboss.bom.eap>
```

Comment:
The version.org.jboss.bom.eap is for defining the jboss-javaee-6.0-with-tools.
If jboss-javaee-6.0-with-tools is really not necessary (I see it is removed), maybe this properties is also redundant?